components/keyboard_shortcuts/keyboard_shortcuts_modal/keyboard_shortcuts_modal.test.tsx

```diff
@@ -2,14 +2,19 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import {shallow} from 'enzyme';
+import {IntlProvider} from 'react-intl';
 
-import {mountWithIntl} from 'tests/helpers/intl-test-helper';
 import KeyboardShortcutsModal from 'components/keyboard_shortcuts/keyboard_shortcuts_modal/keyboard_shortcuts_modal';
 
 describe('components/KeyboardShortcutsModal', () => {
     test('should match snapshot modal', () => {
-        const wrapper = mountWithIntl(
-            <KeyboardShortcutsModal/>,
+        const wrapper = shallow(
```

Comment:
This is kind of messing up the snapshot for this test since it just contains the provider with a `<KeyboardShortcutsModal/>` inside it without telling us anything about how the component renders. Maybe `shallowWithIntl` would be better here so that the snapshot actually contains the results of the render?

Reply:
`shallowWithIntl` does not seem to work with `withIntl` hook. So i had to resort to this method. But indeed the snapshot is kind of messed up. @saturninoabril do you have any idea how can we do this

Reply:
check with `dive()`

Reply:
As discussed, we can either look at using `mountWithIntl` and `dive` together or mocking `useIntl` to fix this. I also filed a ticket to just mock `useIntl` everywhere in the tests

Reply:
Global mocking of `react-intl` would be good, but we should be mindful of where different locales testing is required. But yeah lets discuss that in seperate ticket.
For our current test i have mocked `useIntl` locally and snapshot seems to be a lot cleaner now.